chrome/content/zotero/zoteroPane.js

```diff
@@ -4054,6 +4054,8 @@ var ZoteroPane = new function()
 		
 		// Build menus for each top-level collection of this library
 		let collections = Zotero.Collections.getByLibrary(this.getSelectedLibraryID());
+		let children = selected.getDescendents(true, "collection");
```

Comment:
Shouldn't this be `false`?

Reply:
You're right! Sorry about that, I very much misunderstood what the `nested` param meant.